web/src/App.js

```diff
@@ -15,9 +14,7 @@ function App() {
   }
 
   return (
-    <EthersContextProvider>
-      <ViewState/>
-    </EthersContextProvider>
+    <ViewState/>
```

Comment:
Theres is there a need for VIewState? Can we put whatever is there in the ViewState component into App?

Reply:
yeah not done yet.